scribus/scpaths.cpp

```diff
@@ -66,6 +67,35 @@ ScPaths::ScPaths() :
 	m_shareDir(SHAREDIR),
 	m_templateDir(TEMPLATEDIR)
 {
+
+// Build an appimage and redirect the paths to
+// ones inside the bundle, relative to the executable.
+#ifdef Q_OS_LINUX
+	// Set the application name expliticly.
+	QCoreApplication::setApplicationName("scribus");
```

Comment:
What's the goal of this?  Are you imagining occasions where the default of "executable name" is wrong?

Reply:
I added that because the file configs were set in a different location every time I installed a new version.
I can't remember now (I told @aoloe in irc a while back) but I believe the folders were named scribus-<version> or similar

Reply:
And this fixed it?  It feels weird, I would love to have more details about this… @aoloe ?

Reply:
the details should be somewhere in irc...
The issue is that there was no application name set properly...
we could test this again if you want..